pkg/blockrsync/hasher.go

```diff
@@ -275,6 +306,22 @@ func (f *FileHasher) DeserializeHashes(r io.Reader) (int64, map[int64][]byte, er
 	return blockSize, hashes, nil
 }
 
+func (f *FileHasher) CompareHashHash(rw io.ReadWriter) (bool, error) {
+	f.log.V(5).Info("Comparing hash of hashes", "hash", base64.StdEncoding.EncodeToString(f.hashHash))
+	if n, err := rw.Write(f.hashHash); err != nil {
+		return false, err
+	} else {
```

Comment:
NIT: no need for an `else` statement here

Reply:
Actually I do need the else here, because I am referencing`n` which only exists in the context of the if statement.